common.gypi

```diff
@@ -367,7 +367,6 @@
         'ldflags': [ '-pthread' ],
       }],
       [ 'OS in "linux freebsd openbsd solaris android aix cloudabi"', {
-        'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', ],
```

Comment:
Some of these are also set for mac on L466-L471.

Reply:
Moved them